kratos/includes/convection_diffusion_settings.h

```diff
@@ -82,6 +82,7 @@ class ConvectionDiffusionSettings
 		mis_defined_SurfaceSourceVar=false;
 		mis_defined_ProjectionVar=false;
 		mis_defined_ConvectionVar=false;
+        mis_defined_GradientVar = false;
```

Comment:
It **may be** that the issue is that there is not a default gradient variable defined in the [convection diffusion solver](https://github.com/KratosMultiphysics/Kratos/blob/master/applications/ConvectionDiffusionApplication/python_scripts/convection_diffusion_solver.py#L91-L103). Can you try to add one variable? We had this issue two years ago that one variable was not defined by default. I think it creates problems with the serializer, which is widely used in the MLMC application.

Reply:
Thank you @riccardotosi 
It seems reasonable. I will try it.